ucm2/codecs/rt1318/init.conf

```diff
@@ -1,5 +1,38 @@
 # RT1318 specific switch control settings
 
+If.oneAmpRemap {
```

Comment:
It would be better to define a new default variable like:

```
Define.SpeakerMixerElem "rt1318-1 DAC"
```

and remove this whole If block.